addon/components/object-list-view.js

```diff
@@ -1620,7 +1621,7 @@ export default FlexberryBaseComponent.extend(
       }
 
       let widthCondition = columnsWidthAutoresize && containerWidth > tableWidth;
-      $table.css({ 'width': (columnsWidthAutoresize ? containerWidth : tableWidth) + 'px' });
+      $table.css('cssText', 'width:' + (columnsWidthAutoresize ? containerWidth : tableWidth) + 'px !important' );
```

Comment:
I suggest using [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals), they are more readable than string concatenation.
Why is the previous use of the `css` function worse?

Reply:
1. +
2. to add '! important' for CSS